src/App/Fossa/EmbeddedBinary.hs

```diff
@@ -109,7 +110,7 @@ writeExecutable path content = do
 extractedPath :: PackagedBinary -> Path Rel File
 extractedPath bin = case bin of
   Syft -> $(mkRelFile "syft")
-  Wiggins -> $(mkRelFile "wiggins")
+  Wiggins -> $(mkRelFile $ if SysInfo.os == "mingw32" then "wiggins.exe" else "wiggins")
```

Comment:
I don't think this should be necessary; is there an issue you ran into without the `.exe` extension?

Reply:
I didn't, this may have been premature.

Reply:
Okay cool. Hoping we can avoid it; trying to keep OS-specific code out of things as much as possible 🙂

Reply:
@cnr what makes this unnecessary?

Reply:
I'd be shocked if `process` requires a `.exe` suffix, especially when we use an absolute path to the executable in the `exec` invocation